trail/upload/process_uploads/standardizers/lbt_standardizer.py

```diff
@@ -0,0 +1,48 @@
+"""
+Class that facilitates header metadata translation for Large Binocular Telescope Observatory
+"""
+
+from datetime import datetime, timedelta, timezone
+
+from upload.process_uploads.header_standardizer import HeaderStandardizer
+from upload.models import Metadata
+
+
+__all__ = ["LbtStandardizer", ]
+
+
+class LbtStandardizer(HeaderStandardizer):
```

Comment:
Please add a class docstring and also function docstrings, ideally following the guide here. It's based on numpydoc and has easy-to-follow examples. https://developer.lsst.io/python/numpydoc.html